pkg/app/pipectl/cmd/application/sync.go

```diff
@@ -59,7 +59,7 @@ func newSyncCommand(root *command) *cobra.Command {
 }
 
 func (c *sync) run(ctx context.Context, t cli.Telemetry) error {
-	statuses, err := makeStatuses(c.status)
+	statuses, err := model.DeploymentStatusesFromStrings(c.status)
```

Comment:
nit: not in the scope of this PR but should we rename this `s.status` to `s.statuses` since it's type `[]string` 👀

Reply:
Nice catch! Let me fix them in this too.

Reply:
Fixed them!